### ISSUE TITLE
Pass Through `chunks` arg in `write_zarr` to `layers`

### ISSUE THREAD

**Issue by ilan-gold** [scverse/anndata#523]
Would it be possible to replicate this functionality:

https://github.com/theislab/anndata/blob/2723c9c9e8bc0fe50a9513f292916616a603923b/anndata/_io/zarr.py#L48-L51

in the `layers` as well?

https://github.com/theislab/anndata/blob/2723c9c9e8bc0fe50a9513f292916616a603923b/anndata/_io/zarr.py#L58

Maybe there is a reason not to do this though

**Comment by ivirshup** [comment]
So, I'm a little bit hesitant to define much about the behavior of `chunks` at the moment.  This is because we'd like to integrate `dask` with anndata, where chunks need to be aligned between arrays that are used together. In this instance, it may make more sense if sparse arrays aren't  chunked along the `indices` or `data` but the sparse arrays themselves would be chunked by their cartesian indices (e.g. you'd get chunks of sparse arrays instead of chunks of dense arrays).

I don't see any harm in adding the behavior you suggest right now, but it could change soon. I think there would be value in being able to specify these things separately.

**Comment by ilan-gold** [comment]
@ivirshup I agree that they should be specified separately, especially since they `layers` and `X` might be different types.  Perhaps in order to wrap up #524 as well, we could just add a `sparse_chunks` argument to `write_zarr`?  Then these could both be passed on to the `layers`?

**Comment by flying-sheep** [comment]
@ilan-gold is this issue still relevant?
